addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2513,6 +2513,55 @@ QUnit.test('chat window should remain folded when new message is received', asyn
     );
 });
 
+QUnit.test('with conversation scrolled to end, when composer size changes, conversation should still be scrolled to end', async function(assert) {
+    assert.expect(2);
+
+    // channel that is expected to be found in the messaging menu
+    // with random unique id, needed to link messages
+    this.data['mail.channel'].records.push({ id: 20 });
+    for (let i = 0; i < 10; i++) {
+        this.data['mail.message'].records.push({
+            body: "not empty",
+            model: "mail.channel",
+            res_id: 20,
+        });
+    }
+    await this.start();
+    await afterNextRender(() => document.querySelector(`.o_MessagingMenu_toggler`).click());
+    await this.afterEvent({
+        eventName: 'o-component-message-list-scrolled',
+        func: () => document.querySelector('.o_NotificationList_preview').click(),
+        message: "should wait until channel 20 scrolled to its last message after opening it from the messaging menu",
+        predicate: ({ scrollTop, thread }) => {
+            const messageList = document.querySelector('.o_ThreadView_messageList');
+            return (
+                thread &&
+                thread.model === 'mail.channel' &&
+                thread.id === 20 &&
+                scrollTop === messageList.scrollHeight - messageList.clientHeight
+            );
+        },
+    });
+
+    const messageList = document.querySelector('.o_ThreadView_messageList');
+    assert.strictEqual(
+        document.querySelector(`.o_ThreadView_messageList`).scrollTop,
+        messageList.scrollHeight - messageList.clientHeight,
+        "scrollTop should be at the end."
+    );
+
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "\n");
+    });
+
+    assert.strictEqual(
+        document.querySelector(`.o_ThreadView_messageList`).scrollTop,
+        messageList.scrollHeight - messageList.clientHeight,
+        "scrollTop should still be the same after inserting a new line inside the composer."
```

Comment:
explain that this works because the composer height is automatically adjusting based on content (note: we have a spec that might change that behavior later)